doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-supported-permissions-ref.adoc

```diff
@@ -0,0 +1,97 @@
+[id='security-management-rest-api-supported-permissions-ref_{context}']
+= Supported permissions in {CENTRAL}
+
+The following are available permissions in {PRODUCT}. Administrators use these permissions to allow specific actions to a group, role, or user in {CENTRAL}.
+
+Priority::
+Priority is an integer that defines the precedence of users who are assigned multiple roles or groups. The default value of priority for a new group is `-100`. In {CENTRAL}, you can set an integer value as a priority, which is resolved using the following rules:
++
+--
+.Priority value table
+[cols="50%,50%", frame="all", options="header"]
+|===
+|Integer value
+|Priority
+
+|Between -5 and 0
+|LOW
+
+|Less than -5
+|VERY LOW
+
+|Equals to 0
+|NORMAL
+
+|Between 0 and 5
+|HIGH
+
+|Greater than 5
+|VERY HIGH
+|===
+--
+
+Home Page::
+`homePage` indicates the default landing page for users.
+
+Workbench::
+Workbench consists of the following defined permissions:
++
+--
+[source,json]
+----
+{
+	"editDataObject": true,
+	"plannerAvailable": true,
+	"editGlobalPreferences": true,
+	"editProfilePreferences": true,
+	"accessDataTransfer": true,
+	"jarDownload": true,
+	"editGuidedDecisionTableColumns": true
+}
+----
+--
+
+Pages, Editor, Spaces, and Projects::
+The following are possible values for the permissions based on the resource type:
++
+--
+* `PAGES: read,create,update,delete`
+* `EDITOR: read`
+* `SPACES: read,update,delete,create`
+* `PROJECT: read,create,update,delete,build`
+
+You can use following code to add exceptions to *Pages*, *Editor*, *Spaces*, and *Projects* permissions:
+
+[source,json]
+----
+[{
+	"name": "HomePerspective",
+	"permissions": {
+		"create": true,
+		"read": true,
+		"delete": false
+	}
+}]
+----
+The `name` attribute is an identifier of a resource that you add as an exception. Use the following REST API endpoints to get the list of possible identifiers. The REST API base URL is `http://SERVER:PORT/business-central/rest/`.
+
+* `[GET] /perspectives`: Returns all pages in {CENTRAL}.
+* `[GET] /editors`: Returns all editors in {CENTRAL}.
+* `[GET] /spaces`: Returns all spaces in {CENTRAL}.
+* `[GET] /spaces/{spaceName}/projects`: Returns projects in a specified space.
```

Comment:
Technically these listed descriptions shouldn't end with a period, since incomplete. You disregard that rule either when at least one of the items in the list is complete, or has multiple sentences, in which case you use a period for all, for consistency. That said, neither should most of the endpoint descriptions throughout this chapter and the doc have periods. But they do end in periods on purpose because some of the descriptions are more than one sentence either in the chapter or in other chapters, and in an effort to standardize, periods are used for all endpoint descriptions throhghout.

So you the choice is yours if you want to keep periods here for consistency with that standard throughout the doc, or remove for consistency with itself. Imo, I would remove the periods here because the are reformatted and separate enough anyway that there's not much need to show the consistency. But your call.